api/storage/elasticsearch/elasticsearch.go

```diff
@@ -111,6 +111,8 @@ func (sm *StorageManager) getDynamicMatchQuery(filters map[string]string, operat
 	var mq *elastic.MatchQuery
 	for name, value := range filters {
 		mq = elastic.NewMatchQuery(name, value)
+		// Minimum number of clauses that must match for a document to be returned
```

Comment:
Suggestion, why don't we change the name of the function to DynamicFunction and pass the Query type: match/term and according to this create the array of elastic.Query ?